ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -115,7 +115,7 @@ querySpecification
 
 windowExpression
     : (IDENTIFIER)?
-     ( tumblingWindowExpression | hoppingWindowExpression | sessionWindowExpression)
+     ( tumblingWindowExpression | hoppingWindowExpression | sessionWindowExpression )
```

Comment:
extra space?

Reply:
made it symmetric on either side, and also consistent with other rules.